index.html

```diff
@@ -0,0 +1,157 @@
+                <li class="link-list-item">4</li>
+              </ul>
+            </section>
+          </li>
+          <li class="project-list-item">item 4
+            <section class="feature">
+              <h2 class="title-feature">header</h2>
+              <p class="content-feature">text</p>
+              <ul class="link-list">
+                <li class="link-list-item">1</li>
+                <li class="link-list-item">2</li>
+                <li class="link-list-item">3</li>
+                <li class="link-list-item">4</li>
+              </ul>
+            </section>
+          </li>
+        </ul> 
+        <h2 class="project-list-title">header</h2>
+        <p class="app-feature">text</p>
+        <ul class="app-list">
+          <li class="app-list-item">1</li>
+          <li class="app-list-item">2</li>
+          <li class="app-list-item">3</li>
+        </ul>
+        <p class="app-feature">text</p>
+        <ul class="app-list">
+          <li class="app-list-item">1</li>
+          <li class="app-list-item">2</li>
+          <li class="app-list-item">3</li>
+        </ul>
+      </div> 
+      </section>
+      <div class="site-wrapper">
+      <section class="fourth-section">
+        <h2 class="title-toughts">header</h2>
+        <p class="about-thoughts">text</p>
+      </section>
+      </div>
+    <div>
+      <div class="site-wrapper">
+        <section class="fifth-section">
+          <h2 class="fifth-section-header">header</h2>
+          <p class="fifth-section-text">text</p>
+          <p class="fifth-section-text">text</p>
+          <p class="fifth-section-text">text</p>
+        </section>
+      </div>
+    </div>
+    <div>
+      <div class="site-wrapper">
+        <section class="sixth-section">
+          <h2 class="sixth-section-header">header</h2>
+          <ul class="skills">
+            <li class="skills-item">1</li>
+            <li class="skills-item">2</li>
+            <li class="skills-item">3</li>
+            <li class="skills-item">4</li>
+            <li class="skills-item">5</li>
+            <li class="skills-item">6</li>
+          </ul>
+          <ul class="skills">
+            <li class="skills-item">1</li>
+            <li class="skills-item">2</li>
+            <li class="skills-item">3</li>
+            <li class="skills-item">4</li>
+            <li class="skills-item">5</li>
+            <li class="skills-item">6</li>
+          </ul>
+          <ul class="skills">
+            <li class="skills-item">1</li>
+            <li class="skills-item">2</li>
+          </ul>
+        </section>
+      </div>
+    </div>
+    <div class="site-wrapper">
+      <section class="seventh-section">
+        <h2 class="seventh-section-header">header</h2>
+        <ul class="for-more">
+          <li class="for-more-item">1</li>
+          <li class="for-more-item">2</li>
+          <li class="for-more-item">3</li> 
+        </ul>
+      </section>
+    </div>
```

Comment:
Indentation.

Reply:
I think it's right. Is it possible to check it again?